fpdf/line_break.py

```diff
@@ -41,6 +41,7 @@ class TextLine(NamedTuple):
     text_width: float
     number_of_spaces_between_words: int
     justify: bool
+    trailing_nl: bool
```

Comment:
Maybe this could have a default value of `False` ?

Reply:
Can `NamedTuple`s have default values? 😮 

Since this is not really part of the public API, where convenience would be desired, I think submitting it explicitly is cleaner.
But I don't really mind either way.